tests/test_other.py

```diff
@@ -10004,6 +10004,14 @@ def test_old_makeDynCall_syntax(self):
     err = self.run_process([EMCC, test_file('test_old_dyncall_format.c'), '--js-library', test_file('library_test_old_dyncall_format.js')], stderr=PIPE).stderr
     self.assertContained('syntax for makeDynCall has changed', err)
 
+  # Tests that dynCalls are produced in Closure-safe way in DYNCALLS mode when no actual dynCalls are used
+  @parameterized({
+    'plain': [[]],
+    'asyncify': [['-sASYNCIFY']],
+    'asyncify_bigint': [['-sASYNCIFY', '-sWASM_BIGINT']]})
+  def test_closure_no_dynCalls_generated(self, args):
```

Comment:
Maybe just call this `test_closure_safe` or `test_closure_error_free` .. there could be other configurations we want to test are also closure safe .. and its not just dynCalls but any undefined symbol that could be problem right?  (i.e. missing library deps or just missing symbols in non-taken branches).